src/app/applications/datatarget/fiware/fiware-detail/fiware-detail.component.html

```diff
@@ -31,6 +31,24 @@ <h3>{{ 'DATATARGET.DETAILS' | translate }}</h3>
               <p>{{ 'DATATARGET.NO-AUTHORIZATIONHEADER' | translate }}</p>
             </ng-template>
 
+            <p><strong>{{ 'DATATARGET.TOKENENDPOINT' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.tokenEndpoint; else showNoTokenEndpointText">{{datatarget.tokenEndpoint}}</pre></code>
+            <ng-template #showNoTokenEndpointText>
+              <p>{{ 'DATATARGET.NO-TOKENENDPOINT' | translate }}</p>
+            </ng-template>
+
+            <p><strong>{{ 'DATATARGET.CLIENTID' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.clientId; else showNoClientIdText">{{datatarget.clientId}}</pre></code>
+            <ng-template #showNoClientIdText>
+              <p>{{ 'DATATARGET.NO-CLIENTID' | translate }}</p>
+            </ng-template>
+
+            <p><strong>{{ 'DATATARGET.CLIENTSECRET' | translate }}</strong></p>
+            <code><pre *ngIf="datatarget.clientSecret; else showNoClientSecretText">{{''.padStart(20, '*')}}</pre></code>
```

Comment:
Hiding the secret is good but you can view the network request and see it. Make sure that the API omits it when fetching the data target and handle it appropriately when saving the data target